src/main/groovy/edu/oregonstate/mist/restpr/api/Season.groovy

```diff
@@ -0,0 +1,28 @@
+package edu.oregonstate.mist.restpr.api
+
+/**
+ * Created by georgecrary on 7/13/15.
+ */
+class Season {
+
+  Integer season_id
+  String  community_name
+  String  cycle_format
+  String  cycle_count
+  Integer elo_default_seed
+  Integer year
+
+  public Season(){
+    // Jackson deserialization
+  }
+
+  Season(Integer season_id , String community_name , String cycle_format , String cycle_count , Integer elo_default_seed , Integer year) {
+    this.season_id = season_id
+    this.community_name = community_name
+    this.cycle_format = cycle_format
+    this.cycle_count = cycle_count
+    this.elo_default_seed = elo_default_seed
+    this.year = year
+  }
+}
```

Comment:
the only call to `Season()` in this pull request is in `SeasonMapper.groovy`, in which you call the default constructor and then initialize the object with a `with {}` block. if you replace those calls with a single groovy "map constructor", you would be able to delete the explicit constructors in this class.